testing/xmrig/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Mainteiner: Oleg Titov <oleg.titov@gmail.com>
+
+pkgname=xmrig
+pkgver=2.8.3
+pkgrel=0
+pkgdesc="XMRig is a high performance Monero (XMR) miner"
+url="https://xmrig.com/"
+arch="all"
+license="GPL-3.0"
+options="!check" # No test suite
+depends_dev=""
+depends="cmake libuv-dev libmicrohttpd-dev openssl-dev"
+makedepends="libmicrohttpd libuv"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/$pkgname/archive/v$pkgver.tar.gz"
```

Comment:
Please replace it with `source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/xmrig/archive/v$pkgver.tar.gz`
It helps to navigate